.cirrus.yml

```diff
@@ -23,7 +25,10 @@ task:
       install_script: pkg install -y ${PKG_COMMON} libressl
       configure_script: ./configure ${CONFIGURE_COMMON} --ssl
     - name: FreeBSD / GnuTLS
-      install_script: pkg install -y ${PKG_COMMON} gnutls
+      install_script:
+        - pkg install -y ${PKG_COMMON} gnutls
+        - git clone --depth 1 https://github.com/neomutt/neomutt-test-files.git ${NEOMUTT_TEST_DIR}
+        - (cd ${NEOMUTT_TEST_DIR} && ./setup.sh)
```

Comment:
Isn't it doing the thing only for the GnuTLS branch?

Reply:
Ah!
I deleted the other branches, whilst testing, to reduce the load on Cirrus.
When I restored them, I forgot to add the testing code to the others.

But... do we need it?
We don't test the OpenSSL/GnuTLS/LibreSSL nor the Curses/Slang code, so there probably isn't any point repeating the tests.

Reply:
Very true - the question is then which branch is the default. But it doesn't matter